### ISSUE TITLE
Support hybrid in AuthenticatorTransport

### ISSUE THREAD

**Issue by moubctez** [kanidm/webauthn-rs#219]
### I did this

A front-end application I've got is using `@github/webauthn-json/browser-ponyfill` and that sends `hybrid` as a value for `transport`. In particular, that is happening when I use Touch ID.

I see `Hybrid` is commented-out in `pub enum AuthenticatorTransport`. Would it be possible to add/uncomment it?

Here is a related issue for `@github/webauthn-json/browser-ponyfill`:

https://github.com/github/webauthn-json/issues/67

### I expected the following

Hybrid transport is supported.

### What actually happened

There is an error:

```
Error("unknown variant `hybrid`, expected one of `usb`, `nfc`, `ble`, `internal`", line: 1, column: 658)).
```

### Version (and git commit)

webauthn-rs v0.4.7

**Comment by Firstyear** [comment]
Thanks, I have put in a PR for this. It was left commented because it's part of the draft level 3 specification and isn't "standardised" yet (it's already been renamed from caBLE to hybrid). 

What device were you using that had hybrid as a transport?

**Comment by moubctez** [comment]
> 

That's on Safari 16.1 on macOS 12.6.

**Comment by Firstyear** [comment]
@moubctez It will be resolved in 0.4.8 that I am releasing today. :) let us know if there is anything else!
